src/main/java/racingcar/domain/RacingService.java

```diff
@@ -0,0 +1,51 @@
+package racingcar.domain;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import racingcar.domain.dto.CarDTO;
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingService {
+  public RacingService() {}
+  public List<CarDTO> setCarListName(List<String> carList){
+    List<CarDTO> list = new ArrayList<>();
+    for(String name : carList){
+      CarDTO car = new CarDTO();
+      car.setCar(name);
+      list.add(car);
+    }
+    return list;
+  }
+  public void gamePlay(int tryCycleChoice,List<CarDTO> racingCarList){
+    for (int i = 0; i < tryCycleChoice; i++) {
+      for(CarDTO car : racingCarList){
+        int randomNumber = Randoms.pickNumberInRange(1,9);
+        if(randomNumber > 3){
+          car.setMove(1 + car.getMove());
+        }
+        StringBuilder movePoint = new StringBuilder();
+        for (int j = 0; j < car.getMove(); j++) {
+          movePoint.append("-");
+        }
+        System.out.print(car.getCar() + " : ");
+        System.out.println(movePoint);
```

Comment:
```suggestion
        System.out.println(car.getCar() + " : " + movePoint);
```
표준출력은 비용이 크니 한 번에 출력하는 것은 어때요?

Reply:
오..! 너무 좋아요!